runtime/gc_base/IndexableObjectAllocationModel.hpp

```diff
@@ -124,6 +124,10 @@ class MM_IndexableObjectAllocationModel : public MM_JavaObjectAllocationModel
 	 */
 	bool initializeAllocateDescription(MM_EnvironmentBase *env);
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+	void *doubleMapArraylets(MM_EnvironmentBase *env, J9Object *objectPtr);
```

Comment:
Usually we provide formal function / calling parameters description in declaration in .hpp files. Please add

Reply:
This comment is good. I would just add `env` to formal parameters description as well.  Use something like "thread GC Environment".